Compiler warnings

```
g++ -g -Wall -O2 iptstate.cc -o iptstate -lncurses -lnetfilter_conntrack 
iptstate.cc: In function ‘WINDOW* start_curses(flags_t&)’:
iptstate.cc:539:7: warning: variable ‘y’ set but not used [-Wunused-but-set-variable]
   int y, x;
       ^
iptstate.cc: In function ‘void switch_scroll(flags_t&, WINDOW*&)’:
iptstate.cc:632:10: warning: variable ‘y’ set but not used [-Wunused-but-set-variable]
   int x, y;
          ^
iptstate.cc: In function ‘void get_input(WINDOW*, std::string&, const string&, const flags_t&)’:
iptstate.cc:679:10: warning: variable ‘y’ set but not used [-Wunused-but-set-variable]
   int x, y;
          ^
```

As seen on my Ubuntu 14.04 system running `g++ (Ubuntu 4.8.4-2ubuntu1~14.04.3) 4.8.4`

--- a/iptstate.cc
+++ b/iptstate.cc
@@ -920,7 +920,7 @@ void delete_state(WINDOW *&win, const tentry_t *entry, const flags_t &flags)
   if (ret < 0) {
     string msg = "Failed to delete state: ";
     msg += strerror(errno);
-    c_warn(win, msg.c_str(), flags);
+    c_warn(win, msg, flags);
   }
 
 }

